www/src/components/TokenPlayground/lib/examples/figma-sds.ts

```diff
@@ -0,0 +1,1565 @@
+// MIT License
+
+// Copyright (c) 2024 Figma
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// JSON.stringify() stores these in memory as strings, but declaring them as
+// objects prevents simple typos and errors
+export default {
+  'sds.resolver.json': JSON.stringify({
```

Comment:
Note on these files: these are in `.ts` files because they’re way more portable. Ironically, if these were `.json` files, we’d have to make sure everything is imported and bundled correctly, which is more work (because we probably wouldn’t want 1 network request per `.json` file, we’d be bundling, but there’d be more indirection in the setup). Since we need all these at runtime, and the user is editing in-browser, just doing the work of putting them in a TypeScript object is more straightforward.